packages/website-eslint/build.ts

```diff
@@ -31,9 +31,7 @@ function createResolve(
   join: string,
 ): esbuild.OnResolveResult {
   const resolvedPackage = requireResolved(targetPath + '/package.json');
-  return {
-    path: path.join(resolvedPackage, '../src/', join),
-  };
+  return { path: path.join(resolvedPackage, '../src/', join) };
```

Comment:
[Style] This looks like an unrelated styling change. Could you please revert it and any others? It's harder to review PRs that have unnecessary whitespace changes - especially these larger, more intricate PRs.

Reply:
sorry about that! prettier is annoyingly un-opinionated about objects on one line vs multiple lines and was conflicting with my editor